src/chainstate/stacks/mod.rs

```diff
@@ -871,8 +871,10 @@ pub mod test {
             ]);
         }
 
+        let stx_address = StacksAddress { version: 1, bytes: Hash160([0xff; 20]) };
         let tx_payloads = vec![
-            TransactionPayload::TokenTransfer(StacksAddress { version: 1, bytes: Hash160([0xff; 20]) }, 123, TokenTransferMemo([0u8; 34])),
+            TransactionPayload::TokenTransfer(
```

Comment:
Can you please make this test exhaustive?  I'd like to make sure we test with both variants of `PrincipalData`, since this code is used in other chainstate tests.

Reply:
Sure